alerter.c

```diff
@@ -1,16 +1,22 @@
 #include <stdio.h>
 #include <assert.h>
 
+#define PRODUCTIONCODE_ON   1
+#define PRODUCTIONCODE_OFF  0
+#define ENABLE_PRODUCTION_CODE  PRODUCTIONCODE_OFF

```

Comment:
Production code should not be aware of test code. Test code should be abstracted neatly via an interface or a function pointer.
There is a danger of accidental execution of test code in field!!

Reply:
Yeah, I will note it.
Correction done.